uaclient/contract.py

```diff
@@ -103,18 +104,20 @@ def request_resources(self) -> "Dict[str, Any]":
         )
         return resource_response
 
-    def request_aws_contract_token(self, pkcs7: str):
-        """Requests contract token for auto-attach images on AWS.
+    def request_auto_attach_contract_token(
+        self, instance: clouds.AutoAttachCloudInstance
```

Comment:
We only pass this as a keyword argument, so may as well:

```suggestion
        self, *, instance: clouds.AutoAttachCloudInstance
```